nsv13/code/modules/overmap/types/syndicate.dm

```diff
@@ -64,26 +64,6 @@
 
 //AI Versions
 
-/obj/structure/overmap/syndicate/ai/Initialize(mapload)
-	. = ..()
-	name = "[name] ([rand(0,999)])"
-
-/obj/structure/overmap/hostile/ai/fighter/Initialize()
-	. = ..()
-	name = "[name] ([rand(0,999)])"
-
```

Comment:
This type doesn't overlap with the one before and thus shouldn't be removed.

Reply:
you right 👍